packages/cli/src/helpers/simulate-device-options.ts

```diff
@@ -0,0 +1,38 @@
+import { Emulation } from 'chrome-debugging-client/dist/protocol/tot';
+import { readFileSync } from 'fs';
+import { resolve } from 'path';
+
+export interface EmulateDeviceSetting extends Emulation.SetDeviceMetricsOverrideParameters {
+  name: string;
+  typeable: string;
+  userAgent: string;
+}
+
+/**
+ * "name" is expected to be a titlecased string. We want something the user can type easily so the passed string
+ * is converted into lowercased words dasherized. Any extra "/" will also be removed.
+ *
+ * @param str - String to be converted to dasherized case
+ */
+function convertToTypable (name: string): string {
```

Comment:
lets bump this function out of "simulate-device-options" and into https://github.com/TracerBench/tracerbench/blob/master/packages/cli/src/helpers/utils.ts and import here

Reply:
done